Hangfire.sln

```diff
@@ -49,6 +48,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = ".nuget", ".nuget", "{F710C9
 		.nuget\packages.config = .nuget\packages.config
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebApplication1", "..\Documents\Visual Studio 2015\Projects\WebApplication1\WebApplication1\WebApplication1.csproj", "{BAE317F5-F6D3-462A-8866-9E50FA00B78B}"
```

Comment:
Again?!?!? @Anthon123 I think every time I find this in a pull request you should put £1 in the charity box.